doc/classes/@GlobalScope.xml

```diff
@@ -361,42 +361,40 @@
 			<return type="Variant" />
 			<param index="0" name="x" type="Variant" />
 			<description>
-				Rounds [param x] downward (towards negative infinity), returning the largest whole number that is not more than [param x]. Supported types: [int], [float], [Vector2], [Vector3], [Vector4].
+				Rounds [param x] downward (towards negative infinity), returning the largest whole number that is not more than [param x]. Supported types: [int], [float] (real), [Vector2], [Vector3], [Vector4].
```

Comment:
As far as I remember, all references to `real` were removed in 4.0?

Reply:
Honestly would've stripped them, but because they've been in the Documentation for so long and no one has caught them, I'd be surprised if it isn't an obscure design decision to keep some of them.